dist/css/style.css

```diff
@@ -0,0 +1,219 @@
+html, body {
+  margin: 0;
+  padding: 0;
+  font-family: "Calibri", "Arial", "Helvetica Neue", sans-serif;
+  overflow: hidden;
+}
+
+.page {
+  display: grid;
+  column-gap: 8px;
+  padding: 8px;
+  box-sizing: border-box;
+  height: 100vh;
+  width: 100vw;
+  grid-template-columns: auto 20em;
+  grid-template-rows: 3em auto 3em;
+  grid-template-areas:
+    "header header"
+    "canvas sidebar"
+    "footer footer";
+}
+
+header {
+  grid-area: header;
+  display: flex;
+}
+
+@font-face {
+  font-family: 'Copperplate';
+  src: url('/fonts/Copperplate V1.woff2')  format('woff');
+}
+
+@font-face {
+  font-family: 'OPTICopperplate';
+	 src: url('/fonts/OPTICopperplate.woff2')  format('woff');  
+}
+
+header h1 {
+  margin: 0;
+  flex-grow: 1;
+  font-size: 1.0em;
+  text-transform: uppercase;
+	font-weight:500;
```

Comment:
Hmm, I guess I should enable Prettier for CSS, there's some inconsistent indentation here that would be nice to clean up.  I will add it to the main branch soon!